fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/impl/tool/attribute/ToolHandlers.java

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.tool.attribute;
+
+import java.util.Arrays;
+import java.util.List;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.ShearsItem;
+import net.minecraft.item.ToolItem;
+import net.minecraft.item.ToolMaterial;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.TypedActionResult;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.tool.attribute.v1.DynamicAttributeTool;
+import net.fabricmc.fabric.api.tool.attribute.v1.FabricToolTags;
+
+/**
+ * Entrypoint to register the default tool handlers.
+ */
+public class ToolHandlers implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		ToolManagerImpl.general().register(new ModdedToolsModdedBlocksToolHandler());
+		ToolManagerImpl.general().register(new VanillaToolsModdedBlocksToolHandler());
+		ToolManagerImpl.general().register(new VanillaToolsVanillaBlocksToolHandler());
```

Comment:
Maybe make all these tool handlers their own classes?

Would clean up the nearly 70% of this class that is inner classes. 

Maybe to a `v1.handler` package?